.travis.yml

```diff
@@ -6,9 +6,17 @@ node_js:
     - "7"
     - "8"
 sudo: false
-script: "npm test && npm run docs"
+branches:
+    only:
+    - master
+
+# Run npm test always
+# Run npm docs only when its the master branch build and node 8
+script:
+    - "npm test"
+    - 'if [[ "$TRAVIS_PULL_REQUEST" = "false" && "$TRAVIS_BRANCH" = "master" && "$node_js" = "8" ]]; then npm run docs; fi'
```

Comment:
What's the point of running `npm run docs` on master? I had assumed that we were running `npm run docs` to make sure the docs can be built without errors, but if we're waiting until the docs are on master then it would already be too late if they've broken.